src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19877.cs

```diff
@@ -14,8 +14,14 @@ public Issue19877(TestDevice device) : base(device) { }
 	[Category(UITestCategories.Border)]
 	public void BorderRoundRectangleWithImage()
 	{
+#if ANDROID
+		var borderElement = App.WaitForElement("TestBorder");
+
+		VerifyScreenshot(borderElement);
+#else
 		App.WaitForElement("OasisImage");
 
 		VerifyScreenshot();
+#endif
```

Comment:
This does not work on any other platform? Looks to be a regular screenshot with a crop?